pallets/subtensor/tests/children.rs

```diff
@@ -453,6 +485,10 @@ fn test_do_revoke_child_singular_success() {
     });
 }
 
+// 13: Test revoking a child in a non-existent network
+// This test verifies that attempting to revoke a child in a non-existent network results in an error:
+// - Attempts to revoke a child in a network that doesn't exist
+// - Checks that the appropriate error is returned
 // SKIP_WASM_BUILD=1 RUST_LOG=info cargo test --test children -- test_do_revoke_child_singular_network_does_not_exist --exact --nocapture
 #[test]
 fn test_do_revoke_child_singular_network_does_not_exist() {
```

Comment:
can't see any child revoke.

Reply:
Setting an empty vec for children is the revoke operation.